src/truffle/approve-tokens.js

```diff
@@ -193,12 +202,18 @@ async function approveAndDisapprove (contractsInfo, params, tokensToApprove, tok
 }
 
 async function loadContractsInfo () {
-  const Proxy = artifacts.require('DutchExchangeProxy')
-  const DutchExchange = artifacts.require('DutchExchange')
+  let { whitelist } = argv
+  let contractName
+  if (!web3.isAddress(whitelist)) {
+    const WhiteListingContract = artifacts.require(whitelist)
+    const whiteListingContract = await WhiteListingContract.deployed()
+    contractName = whitelist
+    whitelist = whiteListingContract.address
+  }
 
   // Get contract examples
-  const proxy = await Proxy.deployed()
-  const dx = DutchExchange.at(proxy.address)
+  const TokenWhitelist = artifacts.require('TokenWhitelist')
+  const white = await TokenWhitelist.at(whitelist)
```

Comment:
heheh, I would call it tokenWhitelist instead if white, to not be confused with the color :)

Reply:
I know, I should have called it beige.

Reply:
💃